packages/ui/client/components/BrowserIframe.vue

```diff
@@ -61,30 +70,54 @@ function changeViewport(name: string) {
         @click="changeViewport('tablet')"
       />
     </div>
-    <div flex-auto overflow-auto>
-      <div id="tester-ui" class="flex h-full justify-center items-center font-light op70" style="overflow: auto; width: 100%; height: 100%" :data-viewport="viewport">
+    <div class="grid scrolls place-items-center" style="height: calc(100vh - 84px)">
+      <div
+        id="tester-ui"
+        class="flex font-light op70"
+        :class="resize ? 'resizing': undefined"
+        :data-viewport="viewport"
+      >
         Select a test to run
       </div>
     </div>
   </div>
 </template>
 
 <style>
+#tester-ui.resizing iframe {
+  pointer-events: none;
+}
+[data-viewport="custom"] {
+  padding: 11px;
```

Comment:
why do you need margins and paddings?

Reply:
I intentionally _removed_ all paddings

Reply:
It is about the 11px on the left resizer (avoid overlap)

Reply:
I can remove both margin and padding, shouldn't be a problem

Reply:
There is no overlap, the resizer appears on top of the panel